ecl/hql/hqlmanifest.cpp

```diff
@@ -198,6 +198,64 @@ void ResourceManifest::loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+void addSecretToJfrog(IPropertyTree &item)
+{
+    StringBuffer password;
+    const char *user = item.queryProp("@jfrogUser");
+    getSecretValue(password, "jfrog", user, "password", true);
```

Comment:
@afishbeck Do you think this is reasonable to add a new category to access jfrog secrets?  Are we going to continue to have new secret categories?
Would it be better to have an "external" category with a key of "jfrog:<user>"?  What other alternatives are there.  The only other similar example at the moment is "git", but maybe we will end up with docker, or other examples.
Thoughts?